ext/search_kit/ang/crmSearchDisplayBatch/crmSearchDisplayBatch.component.js

```diff
@@ -116,6 +116,7 @@
           display: this.display,
           rowCount: this.newBatch.rowCount,
           targets: this.newBatch.targets,
+          label: this.newBatch.name,
```

Comment:
@larssandergreen I find it confusing that you're calling the local variable 'name' but storing it to 'label.